tests/fft.cpp

```diff
@@ -29,6 +29,31 @@ struct DFDim : ddc::PeriodicSampling<Kx>
 {
 };
 
+template <typename X, std::size_t Nx>
+static void test_fourier_mesh()
```

Comment:
```suggestion
template <typename X>
static void test_fourier_mesh(std::size_t Nx)
```

Maybe too much to pass `Nx` as a template parameter in this case. This is necessary when we want to generate discrete dimensions on the same continuous dimensions in the same run.